app/controllers/partners/family_requests_controller.rb

```diff
@@ -41,7 +41,10 @@ def create
       if create_service.errors.none?
         redirect_to partners_request_path(create_service.partner_request), notice: "Requested items successfully!"
       else
-        redirect_to new_partners_family_request_path, error: "Request failed! #{create_service.errors.map { |error| error.message.to_s }}}"
+        # Save the original request data in the flash to repopulate the form upon redirection
+        flash[:children_items] = children_items
+        flash[:error_messages] = create_service.errors.full_messages.join(", ")
```

Comment:
I believe there is a `to_sentence` method that is available that can be used to join a list of records in a sentence structure 

It'll show up as
Error 1, Error 2, and Error 3

as opposed to
Error 1, Error 2, Error 3